backend/dataall/modules/datapipelines/cdk/pivot_role_datapipelines_policy.py

```diff
@@ -50,7 +50,7 @@ def get_statements(self):
                     "cloudformation:DescribeStackResources"
                 ],
                 resources=[
-                    f'arn:aws:cloudformation:*:{self.account}:stack/*/*',
+                    f'arn:aws:cloudformation:*:{self.account}:stack/{self.env_resource_prefix}*/*',
```

Comment:
Isn't this permission already granted in `/dataall/core/environment/cdk/pivot_role_core_policies/cloudformation.py`?

Do we need this permission then for data.all pipelines?

Reply:
@noah-paige i removed this permission in the data pipelines as this is already covered under core policy